[CI] ClusterSettingsIT testLoggerLevelUpdateWithTransientSettings failing

**Build Scans:**
- [elasticsearch-periodic-platform-support #12472 / windows-2025_checkpart1_platform-support-windows](https://gradle-enterprise.elastic.co/s/lkddftvd7fgck)

**Reproduction Line:**
```
gradlew ":server:internalClusterTest" --tests "org.elasticsearch.cluster.settings.ClusterSettingsIT.testLoggerLevelUpdateWithTransientSettings" -Dtests.seed=CB8F0291A93F5002 -Dtests.locale=en-LC -Dtests.timezone=Zulu -Druntime.java=26
```

**Applicable branches:**
8.19

**Reproduces locally?:**
N/A

**Failure History:**
[See dashboard](https://es-delivery-stats.elastic.dev/app/dashboards#/view/dcec9e60-72ac-11ee-8f39-55975ded9e63?_g=(refreshInterval:(pause:!t,value:60000),time:(from:now-7d%2Fd,to:now))&_a=(controlGroupState:(initialChildControlState:('0c0c9cb8-ccd2-45c6-9b13-96bac4abc542':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:task.keyword,order:0,selectedOptions:!(),title:'GradleTask',type:optionsListControl),'4e6ad9d6-6fdc-4fcc-bf1a-aa6ca79e0850':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:className.keyword,order:1,selectedOptions:!(org.elasticsearch.cluster.settings.ClusterSettingsIT),title:'Suite',type:optionsListControl),'144933da-5c1b-4257-a969-7f43455a7901':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:name.keyword,order:2,selectedOptions:!('testLoggerLevelUpdateWithTransientSettings'),title:'Test',type:optionsListControl)))))

**Failure Message:**
```
java.lang.AssertionError: All incoming requests on node [node_s3] should have finished. Expected 0 bytes for requests in-flight but got 200 bytes; pending tasks [[{
  "node" : "node_s3",
  "id" : 136,
  "type" : "transport",
  "action" : "internal:cluster/coordination/commit_state",
  "description" : "",
  "start_time" : "2026-07-11T02:35:45.395Z",
  "start_time_in_millis" : 1783737345395,
  "running_time" : "1.5m",
  "running_time_in_nanos" : 95846004900,
  "cancellable" : false,
  "headers" : { }
}]]
```

**Issue Reasons:**
- [8.19] 1 consecutive failures in step windows-2025_checkpart1_platform-support-windows

**Note:**
This issue was created using new test triage automation. Please report issues or feedback to es-delivery.

--- a/server/src/internalClusterTest/java/org/elasticsearch/cluster/settings/ClusterSettingsIT.java
+++ b/server/src/internalClusterTest/java/org/elasticsearch/cluster/settings/ClusterSettingsIT.java
@@ -574,7 +574,7 @@ public class ClusterSettingsIT extends ESIntegTestCase {
         assertEquals("Unknown level constant [BOOM].", e.getMessage());
 
         try {
-            final Settings.Builder testSettings = Settings.builder().put("logger.test", "DEBUG").put("logger._root", "debug");
+            final Settings.Builder testSettings = Settings.builder().put("logger.test", "ERROR").put("logger._root", "error");
             ClusterUpdateSettingsRequestBuilder updateBuilder = clusterAdmin().prepareUpdateSettings(
                 TEST_REQUEST_TIMEOUT,
                 TEST_REQUEST_TIMEOUT
@@ -582,8 +582,8 @@ public class ClusterSettingsIT extends ESIntegTestCase {
             consumer.accept(testSettings, updateBuilder);
 
             updateBuilder.get();
-            assertEquals(Level.DEBUG, LogManager.getLogger("test").getLevel());
-            assertEquals(Level.DEBUG, LogManager.getRootLogger().getLevel());
+            assertEquals(Level.ERROR, LogManager.getLogger("test").getLevel());
+            assertEquals(Level.ERROR, LogManager.getRootLogger().getLevel());
         } finally {
             ClusterUpdateSettingsRequestBuilder undoBuilder = clusterAdmin().prepareUpdateSettings(
                 TEST_REQUEST_TIMEOUT,

